planner/languages/rust/rust_planner.go

```diff
@@ -17,9 +24,82 @@ func (p *Planner) Name() string {
 }
 
 func (p *Planner) IsRelevant(srcDir string) bool {
-	return false
+	cargoTomlPath := filepath.Join(srcDir, cargoToml)
+	return plansdk.FileExists(cargoTomlPath)
 }
 
 func (p *Planner) GetPlan(srcDir string) *plansdk.Plan {
-	return &plansdk.Plan{}
+	plan, err := p.getPlan(srcDir)
+	if err != nil {
+		fmt.Printf("[WARNING] error in RustPlanner.GetPlan: %s", err)
```

Comment:
Don't print things inside a planner. If you want, you can pass a plan error

Reply:
Ah, I hadn't seen PlanErrors. Will use that.